src/v2/Components/CreditCardInput/CreditCardInput.tsx

```diff
@@ -99,11 +105,19 @@ const FauxInput = styled(Box)<{ error: boolean }>`
       `}
 
     &:hover {
+      color: ${themeGet("colors.black100")};
       border-color: ${themeGet("colors.black60")};
+      ::placeholder {
+        color: ${themeGet("colors.black100")};
+      }
     }
 
     &--focus {
+      color: ${themeGet("colors.black100")};
       border-color: ${themeGet("colors.black60")};
+      ::placeholder {
+        color: ${themeGet("colors.black100")};
+      }
     }
 
     &--invalid {
```

Comment:
It doesn't look like this border color changes upon error?

Reply:
think it's because of the hover/focus border color overrides it